xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -549,14 +549,14 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         // 4. specific per album
         buttons.Add(CONTEXT_BUTTON_SET_CONTENT, 20195);
       }
-      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtist().size() > 0)
+      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtistString().size() > 0)
```

Comment:
A test library would be very useful, we could include some of the odd cases too, Tolriq is right it needs to be based on some real data rather than all empty files.

Meanwhile being a concientious dev, but not a video music user, I have asked on the support forum for some test data or a tester :)